metaflow/plugins/aws/batch/batch.py

```diff
@@ -23,6 +24,9 @@
     DEFAULT_SECRETS_BACKEND_TYPE,
     AWS_SECRETS_MANAGER_DEFAULT_REGION,
     S3_SERVER_SIDE_ENCRYPTION,
+    FAST_BAKERY_URL,
+    FAST_BAKERY_AUTH,
+    FAST_BAKERY_TYPE,
```

Comment:
we can avoid introducing these as env vars here and in @kubernetes with the expectation that using fast bakery may involve using `@secrets` or `@environment` - to configure the URL and auth. The client can try to read from env vars unless explicitly configured with the URL and auth?